Mathlib/Topology/Algebra/Order/Field.lean

```diff
@@ -299,8 +299,10 @@ instance (priority := 100) LinearOrderedSemifield.toHasContinuousInv₀ {𝕜}
   · filter_upwards [Ioi_mem_nhds (inv_lt_one_of_one_lt₀ hx)] with y hy
     exact inv_lt_of_inv_lt₀ (by positivity) hy
 
-instance (priority := 100) LinearOrderedField.toTopologicalDivisionRing :
-    TopologicalDivisionRing 𝕜 := ⟨⟩
+instance (priority := 100) LinearOrderedField.toIsTopologicalDivisionRing :
+    IsTopologicalDivisionRing 𝕜 := ⟨⟩
+
+@[deprecated (since := "2025-03-25")] alias LinearOrderedField.toTopologicalDivisionRing := LinearOrderedField.toIsTopologicalDivisionRing
```

Comment:
I'm not sure if this deprecation is *required*, but I certainly think it is *fine* to have.

Reply:
I agree, but since this instance was explicitely named, I thought it might be worth it adding the deprecation.